test/e2e-framework/utils/utils.go

```diff
@@ -332,6 +332,20 @@ func GetNonReplicationEnabledSC(provisionerName, scName string) *storagev1.Stora
 	return &scObj
 }
 
+// GetReplicationEnabledSCWithMetroMode returns replication enabled StorageClass testing object with Metro mode
+func GetReplicationEnabledSCWithMetroMode(provisionerName, scName, modeParamName string) *storagev1.StorageClass {
+	scObj := storagev1.StorageClass{
+		ObjectMeta:  metav1.ObjectMeta{Name: scName},
+		Provisioner: provisionerName,
+		Parameters: map[string]string{
+			"param": "val",
+			"replication.storage.dell.com/isReplicationEnabled": "true",
+			"replication.storage.dell.com/" + modeParamName:     "Metro",
```

Comment:
I see in some places that METRO is used and in some places Metro is used. I hope there is no confusion.

Reply:
Given string is converted to upper case in both side car and driver, so we should be good.